<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="elcot-PC" name="ProjectMedium.CreateContact" tests="2" failures="0" timestamp="21 Jul 2018 14:04:11 GMT" time="20.299" errors="1">
  <testcase name="createContact" time="0.000" classname="ProjectMedium.CreateContact">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase name="login" time="20.299" classname="ProjectMedium.CreateContact">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at wdMethods.SeMethods.takeSnap(SeMethods.java:339)
at wdMethods.SeMethods.startApp(SeMethods.java:55)
at wdMethods.ProjectMethods.login(ProjectMethods.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- login -->
</testsuite> <!-- ProjectMedium.CreateContact -->
